roms_tools/tests/test_setup/test_boundary_forcing.py

```diff
@@ -58,6 +58,25 @@ def test_boundary_forcing_creation(boundary_forcing_fixture, request):
     assert hasattr(boundary_forcing.ds, "apply_2d_horizontal_fill")
 
 
+def test_boundary_forcing_creation_with_duplicates(boundary_forcing, use_dask):
```

Comment:
```suggestion
def test_boundary_forcing_creation_with_duplicates(boundary_forcing: BoundaryForcing, use_dask: bool) -> None:
```

Don't forget your typehints!